### PR TITLE
Evovle[Doc]: Reference Shopify's Liquid and Rust's regex

### PR DESCRIPTION
**Pull request by taqtiqa-mark**
2 of 3 addressing issues #607 and #609

### CODE REVIEW

File: guide/src/README.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
 # Introduction
 
-> cargo, make me a project
+```rust
+cargo generate /me/a-project
```
Comment (sassman) [inline guide/src/README.md:4]: I'm not so sure that this slogan should be changed within this PR. Seems pretty unrelated to me.

File: guide/src/README.md [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+```
 
 [![Build status](https://github.com/cargo-generate/cargo-generate/workflows/Build/badge.svg)](https://github.com/cargo-generate/cargo-generate/actions?query=workflow%3ABuild+branch%3Amain+)
 [![crates.io](https://img.shields.io/crates/v/cargo-generate.svg)](https://crates.io/crates/cargo-generate)
 [![dependency status](https://deps.rs/repo/github/cargo-generate/cargo-generate/status.svg)](https://deps.rs/repo/github/cargo-generate/cargo-generate)
 
-`cargo-generate` is a developer tool to help you get up and running quickly with a new Rust
-project by leveraging a pre-existing git repository as a template.
+`cargo-generate` is a Rust developer tool to quickly setup any Rust project,
```
Comment (sassman) [inline guide/src/README.md:11]: > is a Rust developer tool

technically it is not a rust specific developer tool, it could be used in several rust unrelated contexts for example templates containing only Markdown or SQL. Hence I'd like to keep that part as open as possible.

File: guide/src/README.md [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
+by automating repetitive or intricate tasks.
+
+Project templates are ***any*** files.  Alternatively, files can leverage
+[Shopify's Liquid](http://liquidmarkup.org/) templates,
+[Rhai](https://docs.rs/rhai/latest/rhai/) scripts and Rust's 
+[regex crate](https://docs.rs/regex/latest/regex/) to add crates,
+features, initializers, etc. to your freshly created Rust project or an
```
Comment (sassman) [inline guide/src/README.md:18]: > to your freshly created Rust project or an existing Rust project.

That again would make it very rust specific, but the `cargo-generate` is not really limited to that.

**Review by sassman — CHANGES_REQUESTED** [review]
Thanks for your contribution, please find my comments inline.

File: guide/src/README.md (hunk 2 above)
Comment (taqtiqa-mark) [reply guide/src/README.md:11]: > technically it is not a rust specific developer tool, 

Hmm, don't you need Cargo to use it? 

OK so you want the PR to remove 'Rust' in the following:

> .. a new Rust project by ...

Comment (sassman) [reply guide/src/README.md:11]: what I'm trying to say is that the changed description does not speak to me so well. Do you really think that it would bring more clarity than it was before?

Comment (sassman) [reply guide/src/README.md:11]: what I'm trying to say is that the changed description does not speak to me so well. Do you really think that it would bring more clarity than it was before?

**Comment by sassman** [comment]
I meanwhile I've merged some changes only related to the references but not related to the description changes.
